tests/unit/testUtapiClient.js

```diff
@@ -17,28 +17,25 @@ const redisLocal = { host: 'localhost', port: 6379 };
 const config = {
     redis: redisLocal,
     localCache: redisLocal,
+    component: 's3',
 };
 
-// Get prefix values to construct the expected Redis schema keys
+// Build prefix values to construct the expected Redis schema keys
 function getPrefixValues(timestamp) {
-    const arr = [];
-    Object.keys(metricTypes).forEach(metric => {
-        if (metricTypes[metric] === undefined) {
-            return;
-        }
-        const name = metricTypes[metric];
-        let type;
-        if (metric === 'bucket') {
-            type = 'buckets';
-        } else if (metric === 'accountId') {
-            type = 'accounts';
-        }
-        arr.push({
-            key: `s3:${type}:${name}`,
-            timestampKey: `s3:${type}:${timestamp}:${name}`,
-        });
-    });
-    return arr;
+    return [
+        {
+            key: 's3:buckets:foo-bucket',
+            timestampKey: `s3:buckets:${timestamp}:foo-bucket`,
+        },
+        {
+            key: 's3:accounts:foo-account',
+            timestampKey: `s3:accounts:${timestamp}:foo-account`,
+        },
+        {
+            key: 's3:service:s3',
```

Comment:
Why not just `s3:service`. If any other service ends up using Utapi, it will be namespaced accordingly, for example `metadata:service`.